packages/core/src/middleware/koa-auth/types.ts

```diff
@@ -4,6 +4,8 @@ type Auth = {
   type: 'user' | 'app';
   id: string;
   scopes: Set<string>;
+  /** If the request is verified by a verification record, this will be set to `true`. */
+  permissionVerified?: boolean;
```

Comment:
identityVerified? Permission is a little bit confusing.

Reply:
updated